scripts/vcf_site_level_filtering_wdl/JointVcfFiltering.wdl

```diff
@@ -237,21 +243,20 @@ task ScoreVariantAnnotations {
 
 		ln -s ~{sep=" . && ln -s " model_files} .
 
-		conda install -y --name gatk dill
-
 		export GATK_LOCAL_JAR=~{default="/root/gatk.jar" gatk_override}
 
 		gatk --java-options "-Xmx~{command_mem}m" \
 			ScoreVariantAnnotations \
 			~{"-L " + interval_list} \
 			-V ~{vcf} \
 			-O ~{basename}.~{mode} \
-			--python-script ~{scoring_python_script} \
+			--model-backend ~{model_backend} \
```

Comment:
Yes, I just went ahead and made the model backend an explicit parameter now. Seems like the amount of argument validation would be the same (e.g., if we implicitly toggled on a custom backend using the presence of a script argument, we'd want to check that a backend argument wasn't specified) and it might make WDLing a bit simpler?